v4.5/Makefile

```diff
@@ -0,0 +1,322 @@
+PROFILE := v4.5
```

Comment:
It can't. I intend to copy this file to each of the profile folders, i.e. 2.0, 3.5, 4.5.1, 4.5.2 etc.

I thought about code sharing but IMHO there's very little that could be shared since each profile is distinct and e.g. System.dll can have different references in 2.0 than 4.5.

The profiles are essentially frozen in time so I don't think we'll have a reason to modify the Makefile ever again.